openreview/api/client.py

```diff
@@ -520,15 +520,15 @@ def batches(items, batch_size=1000):
         if confirmedEmails:
             full_response = []
             for email_batch in batches(confirmedEmails):
-                response = self.session.post(self.profiles_search_url, json = {'emails': email_batch}, headers = self.headers)
+                response = self.session.post(self.profiles_search_url, json = {'confirmedEmails': email_batch}, headers = self.headers)
                 response = self.__handle_response(response)
                 full_response.extend(response.json()['profiles'])
 
             profiles_by_email = {}
             for p in full_response:
-                profile = Profile.from_json(p)
-                if p['email'] in profile.content['emailsConfirmed']:
-                    profiles_by_email[p['email']] = profile
+                profile_confirmed_emails = p.get('confirmedEmails', p['content'].get('emailsConfirmed', []))
```

Comment:
I don't understand why you do this
```python
profile_confirmed_emails = p.get('confirmedEmails', p['content'].get('emailsConfirmed', []))
```
and not just this
```python
profile_confirmed_emails = p['content'].get('emailsConfirmed', [])
```

Reply:
`emailsConfirmed` is what we get from the profiles API now. Once we start hiding the full email addresses, we can not use this anymore. 

In this [PR](https://github.com/openreview/openreview-api/pull/469/files), I'm adding the property `confirmedEmails` to the profile results so we can know which profiles were found for each confirmed email sent in the query. 

This code is doing get by the two properties to be backward compatible.